gammapy/makers/utils.py

```diff
@@ -185,11 +217,26 @@ def _map_spectrum_weight(map, spectrum=None):
     return map * weights.reshape(shape.astype(int))
 
 
+def evaluate_bkg(pointing, ontime, bkg, geom, use_region_center, obstime, d_omega):
```

Comment:
Can you add a complete docstring here?

Reply:
Done.
I also changed the function to be non-public.